embedded-bins/Makefile.variables

```diff
@@ -1,7 +1,7 @@
-alpine_version = 3.16
+alpine_version = 3.17
```

Comment:
Is it required to bump the Alpine version here? If yes, I suggest to properly cherry-pick the upstream PRs:

* #2472
* #2643

Reply:
Done in:
* #3206
* #3214